File: app/src/main/java/email/app/response/SendMessage.java
package email.app.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SendMessage {

  private String sender;
  private String receiver;
  private String subject;
  private String text;



}

File: src/app/common/class/SendMessage.ts
export class SendMessage {

constructor(sender: string,receiver: string,subject: string, text: string) {

  this.sender = sender;
  this.receiver = receiver;
  this.subject = subject;
  this.text = text;

}

  sender: string = "";
  receiver: string = "";
  subject: string = "";
  text: string = "";


}
